makefile

```
@@ -0,0 +1,47 @@
+.PHONY: bash build check check-all down format run shell test help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)